test/common/grpc/json_transcoder_filter_test.cc

```diff
@@ -322,52 +324,41 @@ INSTANTIATE_TEST_CASE_P(
     GrpcJsonTranscoderFilterPrintOptions, GrpcJsonTranscoderFilterPrintTest,
     ::testing::Values(
         GrpcJsonTranscoderFilterPrintTestParam{
-            R"(
-    {
-     "proto_descriptor": "test/proto/bookstore.descriptor",
+            R"({
+     "proto_descriptor": "{{ test_rundir }}/test/proto/bookstore.descriptor",
      "services": ["bookstore.Bookstore"]
-    }
-			                                           )",
+    })",
             R"({"id":"101","gender":"MALE","lname":"Shakespeare"})"},
-        GrpcJsonTranscoderFilterPrintTestParam{R"(
-    {
-     "proto_descriptor": "test/proto/bookstore.descriptor",
+        GrpcJsonTranscoderFilterPrintTestParam{R"({
+     "proto_descriptor": "{{ test_rundir }}/test/proto/bookstore.descriptor",
      "services": ["bookstore.Bookstore"],
      "print_options":{"add_whitespace": true}
-    }
-			                                           )",
+    })",
                                                R"({
  "id": "101",
  "gender": "MALE",
  "lname": "Shakespeare"
 }
 )"},
```

Comment:
this extra new-line can't be collpase since *add_spaces* will output a trailing new-line.